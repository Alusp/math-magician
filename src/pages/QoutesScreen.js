import React from 'react';

function QoutesScreen() {
  return (
    <div className="container mx-auto py-32 flex justify-center items-center">
      <p className="text-3xl">
        Mathematics is not about numbers, equations computations
        <br />
        or algorithms: it is about understanding.
        <span className="font-semibold"> -William Shakespare</span>
      </p>
    </div>
  );
}

export default QoutesScreen;
