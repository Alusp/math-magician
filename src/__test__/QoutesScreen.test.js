import React from 'react';
import renderer from 'react-test-renderer';
import QoutesScreen from '../pages/QoutesScreen';
import '@testing-library/jest-dom';

describe('Test QoutesScreen', () => {
  test('Testing QoutesScreen', () => {
    const tree = renderer.create(<QoutesScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});