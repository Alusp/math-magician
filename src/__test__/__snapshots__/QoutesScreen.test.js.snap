// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test QoutesScreen Testing QoutesScreen 1`] = `
<div
  className="container mx-auto py-32 flex justify-center items-center"
>
  <p
    className="text-3xl"
  >
    Mathematics is not about numbers, equations computations
    <br />
    or algorithms: it is about understanding.
    <span
      className="font-semibold"
    >
       -William Shakespare
    </span>
  </p>
</div>
`;
